demo-transactions/src/main/kotlin/io/holixon/cqrshexagonaldemo/demoparent/transactions/adapter/outbound/account/jpa/AccountOutAdapter.kt

```diff
@@ -5,11 +5,13 @@ import io.holixon.cqrshexagonaldemo.demoparent.transactions.application.port.out
 import io.holixon.cqrshexagonaldemo.demoparent.transactions.domain.model.account.Account
 import io.holixon.cqrshexagonaldemo.demoparent.transactions.domain.model.account.Iban
 import io.holixon.cqrshexagonaldemo.demoparent.transactions.framework.OutAdapter
+import jakarta.transaction.Transactional
+import java.math.BigDecimal
 
 @OutAdapter
-class AccountOutAdapter(
-        private val jpaAccountOutAdapter: JpaAccountOutAdapter,
-        private val accountEntityMapper: AccountEntityMapper
+open class AccountOutAdapter(
```

Comment:
Why is this an open class? Is has no subclasses and also should not have

Reply:
It's because of the @Transactional the class must not be final, so a solution would be to open the class, but checking your other comment if transaction is not a good fit to be on this class, the better solution is to revert it and make it final again.